mmv1/products/ces/Example.yaml

```diff
@@ -0,0 +1,135 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+name: Example
+description: Description
+base_url: projects/{{project}}/locations/{{location}}/apps/{{app}}/examples
+update_mask: true
+self_link: projects/{{project}}/locations/{{location}}/apps/{{app}}/examples/{{name}}
+create_url: projects/{{project}}/locations/{{location}}/apps/{{app}}/examples?exampleId={{example_id}}
+update_verb: PATCH
+id_format: projects/{{project}}/locations/{{location}}/apps/{{app}}/examples/{{name}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/apps/{{app}}/examples/{{name}}
+examples:
+  - name: "ces_example_basic"
+    primary_resource_id: "my-example"
+    vars:
+      example_display_name: 'my-example'
+      app_display_name: 'my-app'
+      example_id: 'example-id'
+      app_id: 'app-id'
+autogen_status: RXhhbXBsZQ==
+parameters:
+  - name: location
+    type: String
+    description: Resource ID segment making up resource `name`. It identifies the resource within its parent collection as described in https://google.aip.dev/122.
+    immutable: true
+    url_param_only: true
+    required: true
+  - name: app
+    type: String
+    description: Resource ID segment making up resource `name`. It identifies the resource within its parent collection as described in https://google.aip.dev/122.
+    immutable: true
+    url_param_only: true
+    required: true
+  - name: exampleId
+    type: String
+    description: |-
+      The ID to use for the example, which will become the final component of
+      the example's resource name. If not provided, a unique ID will be
+      automatically assigned for the example.
```

Comment:
The field is marked required and must always be provided in Terraform; we should update the description to reflect that.